dev-images/vagrant/Vagrantfile

```diff
@@ -16,15 +17,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         libvirt.cpus = 1
         libvirt.cpu_mode = 'host-passthrough'
         libvirt.nested = true
-        
+
         libvirt.management_network_name = 'leapp-devenv-network'
         libvirt.management_network_address = '192.168.124.0/24'
     end
 
-    config.vm.provision "shell", inline: "dnf -y update && dnf -y install docker libvirt"
+    config.vm.provision "shell", inline: "dnf -y upgrade && dnf -y install docker"
 
-    config.vm.provision "shell", inline: "cd #{HOME} && curl -O #{BUILD_SCRIPT_URL} && $SHELL #{BUILD_SCRIPT} 2>&1 | tee #{HOME}/leapp-setup.log", privileged: false
-    config.vm.provision "shell", inline: "systemctl enable docker libvirtd"
-    config.vm.provision "shell", inline: "systemctl start docker libvirtd"
+    config.vm.provision "shell", inline: "dnf install -y git ansible && "\
+        "cd #{HOME} && git clone #{GIT_URL} #{GIT_CLONE_DIR} && "\
+        "cd #{GIT_CLONE_DIR} && git checkout #{GIT_BRANCH} && "\
+        "cd ansible && ansible-playbook leapp_dev.yml 2>&1 | tee #{HOME}/leapp-setup.log"
```

Comment:
IMO:
cd ansible && ansible-playbook leapp_dev.yml

would be better as:

ansible-playbook ansible/leapp_dev.yml

Reply:
I was trying to be consistent with the Dockerfile again, so it is executed as `cd ansible && ansible-playbook leapp_dev.yml`. If the change is desired I will apply it also on the Dockerfile.